index.html

```diff
@@ -61,14 +76,23 @@
                 <li>Famliy Time is a must.</li>
               </ul>
           </section>
+          <section id='bio'>
+            <h2>Bio</h2>
+            <p>Hello there, thank you for checking out my work space. My name is Dan Stineback and I'm a web developer....well aspiring web developer. Web development wasn't always on my radar but has always been something that I loved doing. So I decided to take the leap and become a full time developer. This site is a work in progress but in the coming months will be launched live and out into the world. Until then I hope you enjoy it.</p>
+            </section>
         </section>
+      </article>
     </main>
-    <footer>&copy; 2016 Dan Stineback</footer>
+    <div class="clearfix"></div>
+  <footer>&copy; 2016 Dan Stineback</footer>
   <script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
-  <script src="js/projectData.js" type="text/javascript"></script>
-  <script src="js/projects.js" type="text/javascript"></script>
-  <script src="js/projectView.js" type="text/javascript"></script>
-
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.4/handlebars.js"></script>
+  <script src="scripts/projectData.json"></script>
```

Comment:
Why are you including your JSON file in the HTML? You don't need to do that, that's what the AJAX call is for.